Source/Language/Flow/SetupSequencePhrase.cs

```diff
@@ -40,6 +40,8 @@
 
 using System;
 using System.ComponentModel;
+using System.Reflection;
+using Moq.Properties;
```

Comment:
Please insert a blank line before this one (such that namespace imports are grouped by main namespace).

Reply:
:+1: